dags/challenge_configs.yaml

```diff
@@ -1,54 +1,77 @@
-pegs:
+olfactory-2025-task1:
   synapse_conn_id: "SYNAPSE_ORCA_SERVICE_ACCOUNT_CONN"
   aws_conn_id: "AWS_TOWER_PROD_S3_CONN"
-  revision: "558a3786e1610a0cb802e384ab082f696030809a" # Updated to the latest revision with new params.entry for PEGS
-  challenge_profile: "pegs_challenge_test"
-  tower_conn_id: "PEGS_CHALLENGE_PROJECT_TOWER_CONN"
-  tower_view_id: "syn58942525"
+  revision: "d63e7c8ce7388db8e03651427b499e851f4edcd7"
+  challenge_profile: "olfactory25_challenge_task1"
+  tower_conn_id: "OLFACTORY_CHALLENGE_PROJECT_TOWER_CONN"
+  tower_view_id: "syn66279193"
   tower_compute_env_type: "spot"
-  bucket_name: "pegs-challenge-project-tower-scratch"
-  key: "10days/pegs_challenge"
+  bucket_name: "olfactory-challenge-project-tower-scratch"
+  key: "10days/olfactory_challenge_task1"
   dag_config:
-    schedule_interval: "*/1 * * * *"
-    start_date: "2024-04-09T00:00:00"  # ISO-format date string; will be converted in the DAG factory
+    schedule_interval: "*/3 * * * *"
+    start_date: "2024-04-09T00:00:00+00:00"
+    end_date: "2025-07-21T00:00:00+00:00"
     catchup: false
     default_args:
       retries: 2
     tags:
       - "nextflow_tower"
 
-olfactory-2025-task1:
+olfactory-2025-task2:
   synapse_conn_id: "SYNAPSE_ORCA_SERVICE_ACCOUNT_CONN"
   aws_conn_id: "AWS_TOWER_PROD_S3_CONN"
   revision: "d63e7c8ce7388db8e03651427b499e851f4edcd7"
-  challenge_profile: "olfactory25_challenge_task1"
+  challenge_profile: "olfactory25_challenge_task2"
   tower_conn_id: "OLFACTORY_CHALLENGE_PROJECT_TOWER_CONN"
-  tower_view_id: "syn66279193"
+  tower_view_id: "syn66484079"
   tower_compute_env_type: "spot"
   bucket_name: "olfactory-challenge-project-tower-scratch"
-  key: "10days/olfactory_challenge_task1"
+  key: "10days/olfactory_challenge_task2"
   dag_config:
     schedule_interval: "*/3 * * * *"
-    start_date: "2024-04-09T00:00:00"
+    start_date: "2024-04-09T00:00:00+00:00"
+    end_date: "2025-07-21T00:00:00+00:00"
```

Comment:
@vpchung let me know what you think of the `end_date` for the leaderboard/final round tasks 1/2 -

I based these off the timeline [here](https://www.synapse.org/Synapse:syn64743570/wiki/632391)

Reply:
Looks good to me, thank you!